ccla-src/cloudclapp-guide/index.adoc

```diff
@@ -55,6 +55,18 @@ The options are "Hourly", "Monthly" and Annual
 === Applications
 === Environments
 ==== Dashboard
+==== Design
+image:screen_design.png[alt=App Access, width=500px]
+
+You will see a designer tool on this tab to create deployment. You can create a blueprint of design also. You will see that created blueprint on blueprints sections once you create that successfully.
+
+===== Designer Tool
+This is playground where you can design your environment/deployment using services/options from link:#custom_palette[Custom Palette]. Once you select any of infra or resource you will see modal on right side to provide configuration/parameters values. Once you done with design then you can save as a blueprint and then you can deploy that design.
+
+[#custom_palette]
+===== Custom Palette
```

Comment:
could you omit "Custom" and simply say "Palette"?
In the code we call it Custom Palette because we've customised it from the default Palette provided by bpmn.js .
But from user's perspective they don't care about the code and also they could feel they can customise the Palette.
Custom sounds we can customise but